src/providers/Wallet.js

```diff
@@ -13,23 +20,122 @@ function useWalletAugmented() {
 
 // Adds Ethers.js to the useWallet() object
 function WalletAugmented({ children }) {
+  // This new state is neede to avoid showing the user the profile signature request on every network change
+  const [chainReseted, setChainReseted] = useState(false)
```

Comment:
I wonder if this is the proper way to handle this side effect. Let's keep it this way, and keep it in our minds once we migrate to IDX.